Meme/express.js

```diff
@@ -0,0 +1,32 @@
+const fs = require('fs');
+const express = require('express');
+const bp = require('body-parser');
+const app = express();
+const gm = require('gm');
+
+app.listen(3608);
+app.use(express.static('public'));
+app.use(bp.json({limit: "500mb"}));
+
+const response = (req,res) =>{
+  const chatMessage = `${req.body.name}: ${req.body.comment}\n`;
+  const textFilePath = '/home/alopez/garagescript/Meme/public/book.txt';
+
+  fs.appendFile(textFilePath, chatMessage,()=>{});
+  res.send();
+}
+app.post('/send', response);
+
+const resp2 = (req, res) =>{
```

Comment:
same for resp2

Reply:
Actually, this is not a response, this is a url component that handles request and gives back response. So by concept this is not correct, plus the purpose of this function is not super readable